openmc/material.py

```diff
@@ -523,6 +523,95 @@ def remove_element(self, element):
             if element == elm:
                 self._elements.remove(elm)
 
+    def add_material(self, material, percent, percent_type='ao'):
+        """Add a material to the material
```

Comment:
This might be less ambiguous if worded as "Mix a material with this material". I would even consider renaming the method as `mix_material(...)` but that's up to you and any others who wish to comment on this.